### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by syzexecutive** [pqina/filepond#929]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you updated FilePond and its plugins?

- [X] I have updated FilePond and its plugins

### Describe the bug

Incorrect use of **label for=FORM_ELEMENT**
Chrome Issues shows this error. I think it's because FilePond creates a `DIV` with `"filepond--drop-label"` class and puts the label for the file input inside it. Shouldn't the `label` and `input` be together?

### Reproduction

FilePond sets file input class to `filepond--browser` and sets random id for it and creates a `DIV` with `"filepond--drop-label"` class and puts the label for the file input inside it.

![Screenshot 2023-08-19 090827](https://github.com/pqina/filepond/assets/57541909/a03d5939-5560-4088-b399-cb3f2c8e1258)

Issues tab on Chrome:

![Screenshot 2023-08-19 090716](https://github.com/pqina/filepond/assets/57541909/4d74bbba-1c7a-4262-8f2a-bf61322e199c)

### Environment

```markdown
- Device: PC
- OS: Windows 11
- Browser: Chrome 116
```

**Comment by rikschennink** [comment]
It points to an `input`, I don't see what's wrong with that?

**Comment by syzexecutive** [comment]
Yes, it points, but Chrome issues tab shows this error! How can I solve this?
_I have nothing special in FilePond config and configured with basics._

**Comment by rikschennink** [comment]
@syzexecutive Perhaps it's a Chrome bug, I wouldn't worry about it.

**Comment by CelticParser** [comment]
It is not a Chrome bug. `form` and `id` need to match and they do not. This error (recently) shows differently in various browsers.

This causes uploading reliability and accessibility issues.

**Comment by rikschennink** [comment]
@CelticParser But they do seem to match up?

![CleanShot 2023-09-29 at 11 08 56@2x](https://github.com/pqina/filepond/assets/1132575/dcc56963-0f8d-4c09-9554-85a20b2292f4)

**Comment by simonmaass** [comment]
very weird... I get the same error... but not everytime...

<img width="782" alt="image" src="https://github.com/pqina/filepond/assets/32599710/ad137b03-2e08-43a6-91d0-fb09f8ca5268">

**Comment by rikschennink** [comment]
Seems like a bug in Chrome? The `id` is clearly there.
